version.sbt

```diff
@@ -0,0 +1 @@
+ThisBuild / version := "0.6.0"
```

Comment:
So do we want to remove the `-SNAPSHOT` from develop right now? 🤔

Reply:
Fixed.